modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/components/page-accessibility-tool/components/_Occurrence.scss

```diff
@@ -0,0 +1,17 @@
+@import 'atlas-variables';
+
+.page-accessibility-tool__sidebar {
```

Comment:
I prefer that the nomenclature is smaller to avoid noise, for example: `panel-a11y__sidebar`.

Reply:
Makes sense but I was trying to avoid abbrev

Reply:
Yes, it's usually good to avoid it but in these cases we may have a lot of noise in the markup, as a11y is a much more used abbreviation than `accessibility`, I think it's ok, we use it here.